dataactvalidator/scripts/loadFSRS.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import func
+
+from dataactcore.models.baseInterface import databaseSession
+from dataactcore.models.fsrs import FSRSAward
+from dataactvalidator.fsrs import retrieveAwardsBatch
+
+def loadFSRS(minId):
+    """Fetch and store a batch of Award/Sub-Award records, replacing previous
+    entries."""
+    awards = list(retrieveAwardsBatch(minId))
+    ids = [a.id for a in awards]
+    with databaseSession() as sess:
```

Comment:
There is also a getSession() function in BaseInterface that returns the session object.  This seems equivalent, except for the caution above about closing the shared connection after each use.  There is also a small possibility that doing things this way will create a new engine, connection, and session for each usage, I'm not sure that we've really tested the constructor for this case.  If you intended this to create a separate engine, connection, and session, that would require some refactoring of the static variables involved.